src/player/index.ts

```diff
@@ -296,7 +297,20 @@ async function renderCurrentStep() {
 
   const showNavigation = hasPreviousStep || hasNextStep || isFinalStep;
   if (!store.isEditing && showNavigation) {
-    content += "\n\n---\n";
+    if (isAccessibilitySupportOn()) {
+      const lineAndFileInfoLabel =
+        step.line && step.file
+          ? `This step is on line ${step.line} in file ${step.file}`
+          : ``;
```

Comment:
Super nit: this else case can be a `''`, not a backtick / format string
(not blocking)